### DISCUSSION TITLE
[Browser] Unable to start correctly

### DISCUSSION THREAD

**Discussion by znoliver** [AvaloniaUI/Avalonia#11037] [Q&A] [answered]
**Describe the bug**
I am trying to develop a sample using 11.0-preview-6，but my application started with an error：
Uncaught SyntaxError: The requested module './avalonia.js' does not provide an export named 'registerAvaloniaModule'

**Screenshots**
![image](https://user-images.githubusercontent.com/59060212/232404122-a0c1ac5c-56f6-45f9-bb2b-b5517a004d8a.png)

**Desktop (please complete the following information):**

- OS: Ubuntu 22.04 LTS
- Version 11.0-preview-6

**Additional context**
11.0-preview-5 normal operation.

**Comment by timunie** [comment]
I have heard you may need to clean the server cache if you updated to preview6. Can you try it?

**Comment by znoliver** [comment]
> I have heard you may need to clean the server cache if you updated to preview6. Can you try it?

Yes，I clean my project and rebuild it , clean my browser history and cache, but it still reports an error

**Comment by timunie** [comment]
The same or a different?

**Comment by znoliver** [comment, marked as answer]
Sorry, I found out it was my fault！
I created my project with Avalonia.Template 0.10.18.7, it use preview 5, so I manually upgraded to preview 6, but the main.js in AppBundle not updated. It's working fine when I update mian.js.
And when will there be a preview 6 template be available？

**Comment by timunie** [comment]
I thought it is available? You need to update them on your own btw. 

I'll convert this issue into a Q&A now, because I think it's not a bug.
